doc/modules/preprocessing.rst

```diff
@@ -453,7 +453,8 @@ that contain the missing values::
     >>> from sklearn.preprocessing import Imputer
     >>> imp = Imputer(missing_values='NaN', strategy='mean', axis=0)
     >>> imp.fit([[1, 2], [np.nan, 3], [7, 6]])
-    Imputer(axis=0, copy=True, missing_values='NaN', strategy='mean', verbose=0)
+    Imputer(add_missing_indicator=False, axis=0, copy=True, missing_values='NaN',
+        strategy='mean', verbose=0)
```

Comment:
I have changed this after the previous commit failed on doctest and updated based on the actual output when I run the command. I understand pep8 warrants more indentation, but this is documentation and we should be able to give the output as it would if we run the code right ? I have never changed anything related to doctests, so let me know if pep8 guidelines need to be followed here also. Thanks.